js/index.js

```diff
@@ -80,6 +82,14 @@ exports = new (Class(function () {
         });
     });
 
+    pluginOn('SettingsOpened', function () {
+      invokeCallbacks(cb_notif_enable_popup);
```

Comment:
this event is when user press open settings btn this is not for enabled settings